docs/coverage-reporter/uploading-coverage-in-advanced-scenarios.md

```diff
@@ -31,6 +31,9 @@ bash <(curl -Ls https://coverage.codacy.com/get.sh) report \
     -l Java $(find . -name 'jacoco*.xml' -printf '-r %p ')
 ```
 
+!!! note
+    The example above works only on systems that use GNU find with support for the `-printf` action, such as Linux.
+
```

Comment:
@claudiacarpinteiro Should we place this admonition above the example? It's not as clean but the user will definitely notice it. Having it below the example may cause the user to ignore it. Thoughts?

Reply:
I think it would work fine above the example, which is usually the way you prefer, right? 😉 

Also, we could format the "find" as code (the suggestion below is worded for the admonition before the example).

```suggestion
!!! note
    This example works only on systems that use GNU `find` with support for the `-printf` action, such as Linux.

```